backend/app/tasks/llm.py

```
@@ -50,9 +85,14 @@ def llm_routing_worker(
     correlation_id: Optional[str] = None,
     request_id: Optional[str] = None,
     max_cost_cents: int = 0,
+    force_failure: bool = False,
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)